tests/tests.go

```diff
@@ -91,11 +91,11 @@ func removeTestNamespaces() {
 		Should(BeTrue(), "Namespace %s haven't been deleted within the given timeout", TestNamespace)
 }
 
-func CreateVmObject(namespace string, running bool, interfaces []kubevirtv1.Interface, networks []kubevirtv1.Network) *kubevirtv1.VirtualMachine {
+func CreateVmObject(namespace string, strategy kubevirtv1.VirtualMachineRunStrategy, interfaces []kubevirtv1.Interface, networks []kubevirtv1.Network) *kubevirtv1.VirtualMachine {
```

Comment:
I haven't noticed a call to this function with a `strategy` value different from `RunStrategyHalted`.
Please consider dropping the parameter.

Reply:
DONE